src/tree/root.ts

```diff
@@ -1,16 +1,27 @@
 import { useAllUserProps } from "../user-props/";
-import type { Tree, InstanceProps, Breakpoint, Props } from "../db-types";
+import type { Tree, InstanceProps, Breakpoint } from "../db-types";
 import { globalCss, setBreakpoints } from "../stitches";
 import { createElementsTree } from "./create-elements-tree";
 import {
   WrapperComponent,
   type WrapperComponentProps,
 } from "./wrapper-component";
 
+export type InstanceChildrenProps = {
+  id: string;
+  prop: string;
+  value: string;
+  instancePropsId?: string | null;
+};
+
+export type ElementProps = InstanceProps & {
```

Comment:
ComponentProps? because we don't use element as terminology, we use component and instance

```suggestion
export type ComponentProps = InstanceProps & {
```

Reply:
uhh, actually this is also wrong, because this is not exactly what is being passed to a component

Reply:
Its just PropsData or something

Reply:
UserProps will also work with current semantics, because I call that thing later this way